### PR TITLE
Edit UG

### PR DESCRIPTION
**Pull request by amosting**
Added:
- Description of UI
- Delete feature description

Reformat:
- Feature descriptions

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -222,38 +235,38 @@ lines as space characters surrounding line-breaks may be omitted when copied ove
```
Comment (LHeng1) [inline docs/UserGuide.md:235]: Can maybe add a brief description of c/COMMENT as well? For uniformity.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -222,38 +235,38 @@ lines as space characters surrounding line-breaks may be omitted when copied ove
...
+        - Comment: Hardworking
+        - Tags: pastTA, deanslist
+- **`add s/A0251647W n/amanda p/89064678 e/amanda@example.com g/4.3 c/`**
+    - Adds a person with the following information:
         - Student number: 
         - Name: Amanda
         - Phone number: 89064678
         - Email address: amanda@example.com
         - GPA: 4.3
+        - Comment: `None`
```
Comment (LHeng1) [inline docs/UserGuide.md:254]: Using ` ` seems to suggest that it is code, and doesn't feel consistent with the other fields. Maybe a simple dash would be better?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -222,38 +235,38 @@ lines as space characters surrounding line-breaks may be omitted when copied ove
...
         - Email address: johndoe@example.com
         - GPA: 5.0
-        - Tags: past TA, dean’s list
-- **`add s/A0251647W n/amanda p/89064678 e/amanda@example.com g/4.3`**
-    - Adds a person with the following information
+        - Comment: Hardworking
+        - Tags: pastTA, deanslist
+- **`add s/A0251647W n/amanda p/89064678 e/amanda@example.com g/4.3 c/`**
+    - Adds a person with the following information:
         - Student number:
```
Comment (LHeng1) [inline docs/UserGuide.md:249]: Missed out the student number.

**Review by LHeng1 — CHANGES_REQUESTED** [review]
Overall LGTM, just some very small nits to fix.

**Comment by ravern** [comment]
Hey @amosting, how's this going?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -114,7 +115,12 @@ An example is shown below.
 
 **Step 2.** Copy the file to the folder you want to use as the _home folder_ for your TAfinder.
 
+<<<<<<< HEAD
+**Step 3.** Double-click on the `tafinder.jar` file to start the app. 
+
+=======
 **Step 3.** Double-click on the `tafinder.jar` file to start the app.
+>>>>>>> 7ae7963ee74c8b190ecb6d659807beed8c4955cd
```
Comment (LHeng1) [inline docs/UserGuide.md:123]: This seems to be a duplicate line with the conflict message still showing.

**Review by LHeng1 — CHANGES_REQUESTED** [review]
Sorry, just a final nit and it should be fine!

**Review by LHeng1 — APPROVED** [review]
LGTM!

**Pull request merged**